test/fixtures/roles.sql

```diff
@@ -5,3 +5,31 @@ CREATE ROLE postgrest_test_default_role;
 CREATE ROLE postgrest_test_author;
 
 GRANT postgrest_test_anonymous, postgrest_test_default_role, postgrest_test_author TO :USER;
+
+-- reloadable config options for io tests
+ALTER ROLE postgrest_test_authenticator SET pgrst."jwt-aud" = 'https://example.org';
```

Comment:
```suggestion
ALTER ROLE postgrest_test_authenticator SET pgrst."raw-media-types" = 'application/vnd.pgrst.db-config';
```

(jwt-aud is added twice, but raw-media-types missing here)